File: 66-plus-one/plus-one.py
class Solution(object):
    def plusOne(self, digits):
        """
        :type digits: List[int]
        :rtype: List[int]
        """
        result=[]
        x=int("".join(map(str,digits)))
        x+=1
        for i in str(x):
            result.append(int(i))
        return result

File: 66-plus-one/plus-one.java
class Solution {
    public int[] plusOne(int[] digits) {
        int p = digits.length-1;
        int carry = 0;
        if(digits[p]!=9){
            digits[p] = digits[p]+1;
            return digits;
        }
        else{
            while(p>=0){
                if(digits[p]==9){
                    digits[p]=0;
                    p--;
                }
                else{
                    digits[p]=digits[p]+1;
                    return digits;
                }
            }
        }

        int ans[]=new int[digits.length+1];
        ans[0] = 1;
        return ans;

    }
}